atlassian_jwt_auth/contrib/django/utils.py

```diff
@@ -1,7 +1,12 @@
+import logging
+
 from django.conf import settings
 from jwt.exceptions import InvalidIssuerError
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Please move this to under `verify_issuers` because it is currently only used there.